src/style.css

```diff
@@ -0,0 +1,61 @@
+.body {
+  margin: 0;
+}
+
+.container {
+  width: 96px;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
```

Comment:
Try to use margin-right(to stars_star) instead justify content, this should fix the bug in the tests